Dfe.Data.SearchPrototype/Web/Tests/PageObjectModel/HomePage.cs

```diff
@@ -14,6 +15,9 @@ public sealed class HomePage : DocumentObjectModelExtractor
         private const string PrivacyLink = "Privacy";
         private const string MainHeadingClass = "govuk-header__link govuk-header__service-name";
 
+        public By Heading => By.CssSelector("header div div:nth-of-type(2) a");
```

Comment:
Can we avoid using selenium-specific syntax in these page classes? 
e.g. instead of **By Heading**, you can just use the GetHomePageHeading() already there. In your AccessibilitySteps, you would use **_homePage.GetHomePageHeading().Should().NotBeNull();**

Reply:
Thanks. I've always used the Selenium syntax when writing such tests so the approach you're taking is new to me but I am happy to learn more about it.